src/Administration/Resources/app/administration/src/app/init/context.init.spec.js

```diff
@@ -101,7 +101,9 @@ describe('src/app/init/context.init.ts', () => {
         Shopware.Store.get('extensions').addExtension({
             name: 'jestapp',
             baseUrl: '',
-            permissions: [],
+            permissions: {
+                read: ['product_read'],
```

Comment:
Just double checking here, shouldn't that be?

```suggestion
                read: ['product'],
```

Same question for lines 119-121.